inference-engine/src/transformations/src/transformations/common_optimizations/hswish_fusion.cpp

```diff
@@ -142,3 +142,44 @@ ngraph::pass::HSwishFusionWithHSigmoid::HSwishFusionWithHSigmoid() {
     auto m = std::make_shared<ngraph::pattern::Matcher>(mul_pattern, matcher_name);
     register_matcher(m, callback);
 }
+
+NGRAPH_RTTI_DEFINITION(ngraph::pass::HSwishFusionWithClamp, "HSwishFusionWithClamp", 0);
+
+ngraph::pass::HSwishFusionWithClamp::HSwishFusionWithClamp() {
+    MATCHER_SCOPE(HSwishFusionWithClampMul);
+    // Replaces a sub-graph (Clamp(x + 3, 0, 6) * x) with a HSwish * 6.
+    const auto input = ngraph::pattern::any_input();
+    const auto add_constant = ngraph::pattern::wrap_type<ngraph::opset8::Constant>();
+    const auto add = ngraph::pattern::wrap_type<ngraph::opset8::Add>({input, add_constant});
+    const auto clamp = ngraph::pattern::wrap_type<ngraph::op::v0::Clamp>({add});
```

Comment:
Do not use internal versioning. Use opset.

Reply:
fix in a13c1d408f64fbfd608da6be465765a116a9b87e